app/Console/Commands/InitCommand.php

```diff
@@ -49,6 +49,10 @@ public function handle(): void
         $this->comment('Remember, you can always install/upgrade manually following the guide here:');
         $this->info('📙  '.config('koel.misc.docs_url').PHP_EOL);
 
+        if ($this->inNoInteractionMode()) {
+            $this->info('Running in --no-interaction mode');
```

Comment:
How about `Running in no-interaction mode` (without the double hyphen)?

Reply:
ok, modifying in next commit